homeassistant/bootstrap.py

```diff
@@ -281,6 +282,12 @@ def mount_local_lib_path(config_dir: str) -> str:
     Async friendly.
     """
     deps_dir = os.path.join(config_dir, 'deps')
-    if deps_dir not in sys.path:
-        sys.path.insert(0, os.path.join(config_dir, 'deps'))
+    env = os.environ.copy()
```

Comment:
I didn't want to modify the environment permanently. I'm not sure if that matters though, but I thought it safer.